app/src/main/java/openfoodfacts/github/scrachx/openfood/dagger/module/AppModule.java

```diff
@@ -21,6 +22,7 @@
 
 @Module
 public class AppModule {
+    private final static OkHttpClient httpClient = Utils.HttpClientBuilder();
```

Comment:
why it's static?

Reply:
I have not changed this line. It was already static, the change appears due to reformatting the code.